source/agora/node/Node.d

```diff
@@ -160,7 +161,7 @@ public class Node : API
             .map!(item => tuple(item.key, item.value))
             .assocArray();
 
-        this.nominator = new Nominator(this.config.node.key_pair,
+        this.nominator = this.getNominator(this.config.node.key_pair,
```

Comment:
It seems weird to add yet more dependency injection when all we really want to do is to stop signing envelopes for tests involving byzantine nodes.

It would be better to actually just subclass `Node` and use its own "fake" nominator class.

I'll write up a comment in one of the existing issues about dependency injections in the test-suite.